biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -449,15 +453,21 @@ subroutine get_harvest_rate_area (patch_land_use_label, hlm_harvest_catnames, hl
      !  Normalize by site-level primary or secondary forest fraction
      !  since harvest_rate is specified as a fraction of the gridcell
      !  also need to put a cap so as not to harvest more primary or secondary area than there is in a gridcell
+     !  For secondary, also need to normalize by the young/old fraction.
      if (patch_land_use_label .eq. primaryland) then
         if (frac_site_primary .gt. fates_tiny) then
-           harvest_rate = min((harvest_rate / frac_site_primary),frac_site_primary)
+           harvest_rate = min((harvest_rate / frac_site_primary),1._r8)
```

Comment:
1._r8 may remove the whole patch. If you have certain protection for no_comp mode in PatchDyanmicMod.F90 to prevent removal of all primary forest, this makes sense. Otherwise I would suggest to use a less than 1 number.

Reply:
yes, we have a cap on disturbance rates [here](https://github.com/NGEET/fates/pull/1116/files#diff-28c679efbe804905b8b4fac3ccdbb04f6bb3bcd2794fe1a4556d5541e008a750R217)